compute_endpoint/tests/unit/test_endpointmanager_unit.py

```diff
@@ -173,6 +173,7 @@ def test_gracefully_exits_if_in_conflict_or_locked(
 
 def test_sends_metadata_during_registration(conf_dir, mock_conf, mock_client):
     ep_uuid, mock_gcc = mock_client
+    mock_conf.multi_tenant = True
```

Comment:
Golly; I'm curious why this didn't fail before?  Is this detail just ... not tested?  Looks like it is.  What changed?

Reply:
We were manually setting `multi_tenant=True` in the [`EndpointManager.get_metadata`](https://github.com/funcx-faas/funcX/blob/6bed1e18efa9669733cbf5ea24c70787a32cf951/compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py#L145) method.

Reply:
Good call.